tapchannel/aux_traffic_shaper.go

```diff
@@ -193,8 +188,8 @@ func (s *AuxTrafficShaper) PaymentBandwidth(htlcBlob,
 	// never be settled. Other HTLCs that may also call into this method are
 	// not yet registered to the commitment, so we need to account for them
 	// manually.
-	computedLocal, _, decodedView, _, err := ComputeView(
-		localBalance, remoteBalance, lntypes.Local, htlcView,
+	computedLocal, decodedView, err := ComputeLocalBalance(
```

Comment:
For ease of review (for future PRs), I always find it easier to do any refactoring first, before adding new functionality or changing things. That way the refactor commit can be validated with the old mental model and the reviewer simply needs to assure that nothing changes in the behavior. And then often the diff of the actual changes are smaller in the refactored code, so also easier to digest. And the same lines aren't touched twice.